tests/Unit/Domain/Test_Block.cpp

```diff
@@ -291,9 +291,73 @@ void test_block_time_dependent_distorted() {
       translation_distorted_to_inertial_map.get_clone());
   test_move_semantics(std::move(original_block), block_copy);
 }
+
+void test_spherical_shell() {
+  const Block<3> spherical_shell(
+      nullptr, 4, DirectionMap<3, BlockNeighbors<3>>{}, "Shell",
+      std::array{domain::Topology::I1, domain::Topology::S2Colatitude,
```

Comment:
Would it help to define this array in Topology.hpp as `spherical_shell_topology`? It will come up more often

Reply:
possibly, though I'm not sure that is the right place to put it.  We can discuss it at the next tech call.